ddtrace/debugging/_debugger.py

```diff
@@ -210,7 +210,7 @@ def disable(cls):
         atexit.unregister(cls.disable)
         unregister_post_run_module_hook(cls._on_run_module)
 
-        cls._instance.stop()
+        cls._instance.stop(join=join)
```

Comment:
Right! It seems that if we call join on a thread after fork in the child process, this will deadlock in Python < 3.7. It looks like earlier versions of Python are not able to handle a "dead" thread in this case where there is nothing to join effectively.